gio/src/subclass/socket_control_message.rs

```diff
@@ -0,0 +1,266 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::{subclass::prelude::*, translate::*, Cast};
+
+use crate::SocketControlMessage;
+
+pub trait SocketControlMessageImpl: ObjectImpl + Send {
```

Comment:
```suggestion
pub trait SocketControlMessageImpl: ObjectImpl + SocketControlMessageImplExt + Send + Sync {
```

Reply:
Ok for the `Sync` (I mean, I need to think about it, but I trust your judgement that is needed :) )... but on `SocketControlMessageImplExt` this is the opposite of what Bilal suggested yesterday here: https://github.com/gtk-rs/gtk-rs-core/pull/1039#discussion_r1123602116

Reply:
* For `Sync`, either you need `Send` and `Sync`, or you need neither :) Only `Send` is not possible with `GObject`s
* For the `SocketControlMessageImplExt` see my comments and also all (?) the other `Impl` traits in gtk-rs-core

Reply:
As agreed offline, removing `Send` and `Sync` entirely.